### PR TITLE
Rebrand concept docs and guides

### PR DESCRIPTION
**Pull request by marklewin**
## Description

Change Nexmo -> Vonage in the guides and concept docs.

Took care to leave URLs, Nexmo Call Control Objects and Nexmo CLI unchanged.

[DEVX-4717](https://nexmoinc.atlassian.net/browse/DEVX-4717)

### CODE REVIEW

File: _documentation/en/client-sdk/setup/create-your-application.md [hunk 1]
```diff
@@ -8,20 +8,20 @@ navigation_weight: 1
...
 
-* [JSON Web Tokens, JWTs](https://jwt.io/) - Client SDK uses JWTs for authentication. In order for a user to log in and use the SDK functionality, you need to provide a JWT per user. JWTs contain all the information the Nexmo platform needs to authenticate requests, as well as information such as the associated Applications, Users and permissions.
+* [JSON Web Tokens, JWTs](https://jwt.io/) - Client SDK uses JWTs for authentication. In order for a user to log in and use the SDK functionality, you need to provide a JWT per user. JWTs contain all the information the Vonage platform needs to authenticate requests, as well as information such as the associated Applications, Users and permissions.
 
 All of these may be [created by your backend](/conversation/overview). 
-If you wish to get started and experience using the SDK without any implementation of your backend, this tutorial will show you how to do so, using the [Nexmo CLI](https://github.com/Nexmo/nexmo-cli).
+If you wish to get started and experience using the SDK without any implementation of your backend, this tutorial will show you how to do so, using the [Nexmo CLI](https://github.com/Vonage/nexmo-cli).
```
Comment (tbedford) [inline _documentation/en/client-sdk/setup/create-your-application.md:18]: I get a 404 for https://github.com/Vonage/nexmo-cli

File: _documentation/en/concepts/guides/glossary.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ For more information on managing CDRs see the [Redact API documentation](/redact
 
 Nexmo Command Line Interface (CLI) tool. 
 
-Further information can be found in the [CLI repository](https://github.com/Nexmo/nexmo-cli).
+Further information can be found in the [CLI repository](https://github.com/Vonage/nexmo-cli).
```
Comment (tbedford) [inline _documentation/en/concepts/guides/glossary.md:47]: I get a 4040 for https://github.com/Vonage/nexmo-cli

File: _documentation/en/conversation/guides/jwt-acl.md [hunk 3]
```diff
@@ -76,11 +76,11 @@ In the previous section, you can see that the `acl` claim has `paths` object con
 
 It is not necessary to provide users with the permissions to access all of these paths. For instance, if a user was not going to upload or receive push notifications, you could create a JWT without including the `/*/applications/**`or `/*/push/**` paths.
 
-## Nexmo Client Libraries
+## Vonage Client Libraries
 
 ### CLI
 
-Currently, you can use the beta version of the [Nexmo CLI](https://github.com/Nexmo/nexmo-cli/tree/beta) to create a JWT [including the appropriate claims](https://github.com/Nexmo/nexmo-cli/tree/beta#jwt)
+Currently, you can use the beta version of the [Nexmo CLI](https://github.com/Vonage/nexmo-cli/tree/beta) to create a JWT [including the appropriate claims](https://github.com/Vonage/nexmo-cli/tree/beta#jwt)
```
Comment (tbedford) [inline _documentation/en/conversation/guides/jwt-acl.md:83]: 404 for https://github.com/Vonage/nexmo-cli/tree/beta

File: _documentation/en/conversation/guides/jwt-acl.md [hunk 4]
```diff
@@ -89,7 +89,7 @@ nexmo jwt:generate ./private.key sub=jamie exp=$(($(date +%s)+86400)) acl='{"pat
 
 ### Node
 
-The beta version of the [Nexmo Node](https://github.com/Nexmo/nexmo-node/tree/beta#jwt) library can also create a JWT [including the appropriate claims](https://github.com/Nexmo/nexmo-node/tree/beta#jwt)
+The beta version of the [Vonage Node](https://github.com/Vonage/nexmo-node/tree/beta#jwt) library can also create a JWT [including the appropriate claims](https://github.com/Vonage/nexmo-node/tree/beta#jwt)
```
Comment (tbedford) [inline _documentation/en/conversation/guides/jwt-acl.md:92]: 404 for https://github.com/Vonage/nexmo-cli/tree/beta

File: _documentation/en/conversation/guides/jwt-acl.md [hunk 5]
```diff
@@ -76,11 +76,11 @@ In the previous section, you can see that the `acl` claim has `paths` object con
 
 It is not necessary to provide users with the permissions to access all of these paths. For instance, if a user was not going to upload or receive push notifications, you could create a JWT without including the `/*/applications/**`or `/*/push/**` paths.
 
-## Nexmo Client Libraries
+## Vonage Client Libraries
```
Comment (tbedford) [inline _documentation/en/conversation/guides/jwt-acl.md:79]: Vonage Server SDKs

File: _documentation/en/conversation/guides/jwt-acl.md (hunk 4 above)
Comment (tbedford) [inline _documentation/en/conversation/guides/jwt-acl.md:92]: Library -> Server SDK

File: _documentation/en/conversation/guides/jwt-acl.md [hunk 6]
```diff
@@ -106,7 +106,7 @@ const aclPaths = {
   }
 }
 
-Nexmo.generateJwt(PRIVATE_KEY, {
+Vonage.generateJwt(PRIVATE_KEY, {
```
Comment (tbedford) [inline _documentation/en/conversation/guides/jwt-acl.md:109]: Has the class name definitely changed?

File: _documentation/en/conversation/guides/jwt-acl.md [hunk 7]
```diff
@@ -117,14 +117,14 @@ Nexmo.generateJwt(PRIVATE_KEY, {
 
 ### PHP
 
-The current version of the [Nexmo PHP](https://github.com/Nexmo/nexmo-php) library can also create a JWT including the appropriate claims when using the Keypair authentication.
+The current version of the [Vonage PHP](https://github.com/Vonage/nexmo-php) library can also create a JWT including the appropriate claims when using the Keypair authentication.
```
Comment (tbedford) [inline _documentation/en/conversation/guides/jwt-acl.md:120]: Library -> Server SDK

Comment (tbedford) [reply _documentation/en/conversation/guides/jwt-acl.md:120]: https://github.com/Vonage/nexmo-php redirects to https://github.com/Vonage/vonage-php-sdk-core

**Review by tbedford — CHANGES_REQUESTED** [review]

File: _documentation/en/conversation/guides/jwt-acl.md (hunk 6 above)
Comment (marklewin) [reply _documentation/en/conversation/guides/jwt-acl.md:109]: No ... good spot

**Comment by marklewin** [comment]
Adding DO NOT MERGE because I want to check that this PR also addresses the issues raised in [DEVX-3912](https://nexmoinc.atlassian.net/browse/DEVX-3912)

**Comment by marklewin** [comment]
Will deal with DEVX-3912 changes in a new branch. Marking as ready for review/merge.

File: _documentation/en/concepts/guides/authentication.md [hunk 8]
```diff
@@ -142,9 +142,9 @@ An example of generating a JWT for a Client SDK application is as follows:
 nexmo jwt:generate ./private.key sub=MY_USER_NAME exp=$(($(date +%s)+86400)) acl='{"paths":{"/*/users/**":{},"/*/conversations/**":{},"/*/sessions/**":{},"/*/devices/**":{},"/*/image/**":{},"/*/media/**":{},"/*/applications/**":{},"/*/push/**":{},"/*/knocking/**":{}}}' application_id=MY_APP_ID
 ```
 
-More examples of generating JWTs with ACLs can be found in the [Nexmo documentation](/conversation/guides/jwt-acl).
+More examples of generating JWTs with ACLs can be found in the [Vonage documentation](/conversation/guides/jwt-acl).
 
-Further information on the Nexmo CLI can be found in its [repository on GitHub](https://github.com/Nexmo/nexmo-cli).
+Further information on the Nexmo CLI can be found in its [repository on GitHub](https://github.com/Vonage/nexmo-cli).
```
Comment (tbedford) [inline _documentation/en/concepts/guides/authentication.md:147]: 404 for https://github.com/Vonage/nexmo-cli

**Review by tbedford — CHANGES_REQUESTED** [review]

**Review by tbedford — APPROVED** [review]

**Pull request merged**